docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -872,7 +873,10 @@ Stratifications for All:
     - 
   * - Healthcare appointments 
     - sum of healthcare appointments 
-    - Split by type of appointment - follow-up vs emergency vs screening 
+    - Split by type of appointment - follow-up vs emergency vs screening as well as usual age/sex/state/etc.
+  * - Missed follow-up appointments 
+    - sum of missed follow-up appointments 
```

Comment:
Is it acceptable to also stratify missed appointments by age/sex, etc? Would be easier to implement since I need to do it for the other ones

Reply:
Yep - I can clarify that here too